scripts/run_e2e_tests.sh

```diff
@@ -207,14 +207,14 @@ done
 # TODO(bhenning): Figure out if this is a bug with protractor.
 if [ "$RUN_ON_BROWSERSTACK" == "False" ]; then
   if [ "$SHARDING" = "false" ] || [ "$SHARD_INSTANCES" = "1" ]; then
-    $NODE_MODULE_DIR/.bin/protractor core/tests/protractor.conf.js --suite "$SUITE"
+    $NODE_MODULE_DIR/protractor/bin/protractor core/tests/protractor.conf.js --suite "$SUITE"
```

Comment:
(Note to reviewers: Made these changes because the .bin/protractor symlink doesn't seem to exist on my local machine. So I resolved the symlink.)